source/common/quic/envoy_quic_server_session.h

```diff
@@ -115,6 +114,15 @@ class EnvoyQuicServerSession : public quic::QuicServerSessionBase,
   quic::QuicSpdyStream* CreateOutgoingBidirectionalStream() override;
   quic::QuicSpdyStream* CreateOutgoingUnidirectionalStream() override;
 
+  quic::HttpDatagramSupport LocalHttpDatagramSupport() override {
+    // TODO(jeongseokson): Http3 Datagram support should be turned on by returning
+    // quic::HttpDatagramSupport::kRfc once the CONNECT-UDP support work is completed. Returning
+    // a value different from quic::HttpDatagramSupport::kNone in this method will make this
+    // session buffer data until a SETTINGS frame is received (by making
```

Comment:
Thinking about this some more, the only reason to wait until was for when two versions of the datagram draft were supported. So I sent out cl/514590767 to fix that. That means we can remove this comment and wait that CL to land and be merged into Envoy before we merge this PR

Reply:
I guess we do not need a runtime flag anymore after the change you've made. Let me know if you think we still need one.

Reply:
The runtime guard is necessary for enabling visible changes, can you confirm that none are enabled? I.e., is there any request that I can send to Envoy that it will react to differently with this PR?